ui/views.py

```diff
@@ -117,13 +118,21 @@ def edit_vocabulary(request, repo_slug, vocab_slug):
 
 
 @login_required
+@permission_required_or_403(
+    # pylint: disable=protected-access
+    # the following string is "learningresources.import_course"
+    # (unless the Repository model has been moved)
+    '{}.{}'.format(
+        Repository._meta.app_label,
+        RepoPermission.import_course[0]
+    ),
+    (Repository, 'slug', 'repo_slug')
+)
```

Comment:
@Ferdi: we agreed on a comment that describes what it means for the permission string

For the rest I don't think there is another or clearer way to call the api